metricbeat/docs/modules_list.asciidoc

```diff
@@ -9,7 +9,7 @@ This file is generated! See scripts/docs_collector.py
 .1+| .1+|  |<<metricbeat-metricset-aerospike-namespace,namespace>>   
 |<<metricbeat-module-apache,Apache>>     |image:./images/icon-yes.png[Prebuilt dashboards are available]    |  
 .1+| .1+|  |<<metricbeat-metricset-apache-status,status>>   
-|<<metricbeat-module-aws,aws>>  beta[]   |image:./images/icon-yes.png[Prebuilt dashboards are available]    |  
+|<<metricbeat-module-aws,aws>>     |image:./images/icon-yes.png[Prebuilt dashboards are available]    |  
 .1+| .1+|  |<<metricbeat-metricset-aws-ec2,ec2>> experimental[]  
```

Comment:
This is still shown as experimental. The reason is that you have to also add `release: ga` in the metricset fields.yml. The reason is that a module can be GA but not all metricsets inside are.

Reply:
@ruflin Thanks for pointing it out! I pushed a new commit with this change.